src/clr-icons/clr-icons.scss

```diff
@@ -4,7 +4,7 @@
 $clr-icon-size: 16px;
 $clr-icon-color-success: hsl(99, 100%, 26%);
 $clr-icon-color-danger: hsl(9, 100%, 44%);
-$clr-icon-color-warning: hsl(47, 100%, 49%);
+$clr-icon-color-warning: hsl(38, 100%, 38%);
```

Comment:
Doesn't the new color reduce the icon's contrast in dark theme?

Reply:
It does. But the contrast is still 4.41:1. Even better than the contrast in light theme.